packages/scm/src/browser/scm-widget.tsx

```diff
@@ -38,16 +38,16 @@ export class ScmWidget extends BaseWidget implements StatefulWidget {
 
     @inject(ScmService) protected readonly scmService: ScmService;
```

Comment:
please revert this file and only remove `protected` from interesting APIs, fields are already readonly, there is no need for getter methods